book/src/01-04-clearing-the-screen.md

```diff
@@ -0,0 +1,153 @@
+# Clearing the Screen
+
+In this section, you'll learn how to clear the screen and handle window resizes.
+
+## The swap chain
+
+`wgpu` represents the framebuffers that will be presented to the screen with a data structure called a `SwapChain`.
+If you're coming from Vulkan or D3D12, you'll already be familiar with this term.
+Briefly, the `SwapChain` determines the format, size and presentation mode of the frames drawn by `wgpu`.
```

Comment:
might be worth saying something in line with "SwapChain links a surface together with a logical device to draw on it"